config/routes.rb

```diff
@@ -188,6 +188,13 @@
   post "webhooks/helpscout_support_dev", to: "webhook#helpscout_support_dev"
   match "webhooks/helpscout", to: lambda { |_| [204, {}, nil] }, via: :head
 
+  get '/description/:path', to: 'descriptions#edit'
```

Comment:
Ca ne fait pas doublon avec l'entrée dans dossier?

Reply:
Globalement, je ne suis pas sûr de comprendre la plus value par rapport à
```
  resources :dossiers, only: [], param: :path do
    resource :description, only: [:edit, :update]
  end
```

Reply:
Je confirme qu'on a bien deux routes qui mènent vers la même action. Tu as raison, ceci suffit : 

```rb
resources :dossiers, only: [], param: :path do
  resource :description, only: [:edit, :update]
end
```

Ce qui nous donne les routes : 

```
GET   /dossiers/:path/description/edit(.:format) descriptions#edit
PATCH /dossiers/:path/description(.:format)      descriptions#update
```

Et c'est très bien du point de vue de la conception des ressources.

Toutefois, ce qu'on veut, c'est une route facile à lire, à retenir, à communiquer à l'intégrateur·rice. Il me semble que `/description/:path` répond beaucoup mieux à ça que `/dossiers/:path/description/edit`. D'où l'ajout explicite de cette nouvelle route, plus explicite.

Reply:
D'ailleurs je n'avais pas yeux en face des trous : ce n'est pas `dossiers` qu'on devrait avoir ici, mais `procedures`. J'ai corrigé les routes :heavy_check_mark: 

```rb
  get '/description/:path', to: 'descriptions#edit'
  resources :procedures, only: [], param: :path do
    member do
      resource :description, only: [:edit, :update]
    end
  end
```

Reply:
Je trouvais ça pas mal : comme dans `DescriptionsController` on a les deux actions `edit` et `update`, je trouvais logique qu'elles apparaissent ici.

Mais en fait, tu as raison, c'est inutile et de toutes façons on voit l'action `edit` apparaître dans la route `  get '/description/:path', to: 'descriptions#edit'` juste au dessus. Je fais le changement :heavy_check_mark: